app/src/main/java/com/github/libretube/ui/extensions/Activity.kt

```diff
@@ -0,0 +1,22 @@
+package com.github.libretube.ui.extensions
+
+import android.app.Activity
+import android.content.res.Configuration
+import androidx.core.view.WindowCompat
+
+val Activity.isLightTheme: Boolean
+    get() {
+        return when (resources.configuration.uiMode and Configuration.UI_MODE_NIGHT_MASK) {
+            Configuration.UI_MODE_NIGHT_YES -> false
+            Configuration.UI_MODE_NIGHT_NO -> true
+            else -> true
+        }
+    }
+
+fun Activity.setLightStatusBarIcons() {
+    WindowCompat.getInsetsController(window, window.decorView).isAppearanceLightStatusBars = false
+}
+
+fun Activity.setDarkStatusBarIcons() {
+    WindowCompat.getInsetsController(window, window.decorView).isAppearanceLightStatusBars = true
+}
```

Comment:
Looks like the method names should be swapped.

Reply:
They are correct. Here we have to change the statusbar icon color but there's no direct method to change icon color directly. So if we set the statusbar as light then icons will be dark

Reply:
Right, missed that.

Reply:
Yes we can do that. It's just a preference. By doing what you suggested the code is not reusable and using functions make code more readable in my opinion

Reply:
True, but it's not like the code is being used anywhere else in your changeset. If it needs to be used in other places as well, it can be easily refactored.